index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+<input type="text" id="numOfRows" placeholder="number of rows">
+<input type="text" id="whichChar" placeholder="symbol to use">
+<input type="button" onclick="makePyramid()" value ="submit">
```

Comment:
Take a look at the requirements. They state that you should make use of event listeners for click + keypress events in order to call your function.